internal/config/config.go

```diff
@@ -191,10 +191,8 @@ func readEnvVariables(k *koanf.Koanf) {
 			return "alertAcknowledgement.author"
 		case "ALERTACKNOWLEDGEMENT_COMMENTPREFIX":
 			return "alertAcknowledgement.commentPrefix"
-		case "SILENCEFORM_AUTHOR_POPULATE_FROM_HEADER_HEADER":
-			return "silenceForm.author.populate_from_header.header"
-		case "SILENCEFORM_AUTHOR_POPULATE_FROM_HEADER_VALUE_RE":
-			return "silenceForm.author.populate_from_header.value_re"
+		case "AUTHENTICATION_HEADER_VALUE_RE":
+			return "authentication.header.value_re"
```

Comment:
Is `AUTHENTICATION_HEADER_NAME` needed here as well?

Reply:
No, this whole section is only needed because all underscores are converted to dots. So `AUTHENTICATION_HEADER_VALUE_RE` becomes `authentication.header.value.re` while `[...].value_re` is needed.
A better naming convention would avoid this whole problem.